bubblesort.js

```diff
@@ -55,9 +76,18 @@ async function anyEvent(context: any, app: Probot) {
   const { name, payload } = context;
   const action = "action" in payload ? payload.action : 'unknown';
   app.log.info({ event: name, action: action });
+  
+  const command = commands.find((command) => comment.includes(command));
+  if (!command) {
+    return "";
+  }
```

Comment:
/cf refactor this code so that it returns an error message if the command is not null